versions/3.0.md

```diff
@@ -1123,6 +1125,344 @@ application/json:
+      parameters: 
+      - name: username
+        type: string
+        in: path
+        required: true
+      - name: slug
+        type: string
+        in: path
+        required: true
+      - name: pid
+        type: string
+        in: path
+        required: true
+      responses: 
+        204:
+          description: the PR was successfully merged
+components:
+  links:
+    UserRepositories:
+      # returns array of '#/components/definitions/repository'
+      operationId: getRepositoriesByOwner
+      parameters:
+        username: $response.username
+    UserRepository:
+      # returns '#/components/definitions/repository'
+      operationId: getRepository
+      parameters:
+        username: $response.owner.username
+        slug: $response.slug
+    RepositoryPullRequests:
+      # returns '#/components/definitions/pullrequest'
+      operationId: getPullRequestsByRepository
+        params: 
+          username: $response.owner.username
+          slug: $response.slug
+    PullRequestMerge:
+      # executes /2.0/repositories/{username}/{slug}/pullrequests/{pid}/merge
+      operationId: mergePullRequest
+      parameters:
+        username: $response.user.username # Should be $response.author.username?
+        slug: $response.repository.slug
+        pid: $response.id
+  definitions: 
+    user: 
+      type: object
+      properties: 
+        username: 
+          type: string
+        uuid: 
+          type: string
+    repository: 
+      type: object
+      properties: 
+        slug: 
+          type: string
+        owner: 
+          $ref: '#/components/definitions/user'
+    pullrequest: 
+      type: object
+      properties: 
+        id: 
+          type: integer
+        title: 
+          type: string
+        repository: 
+          $ref: '#/components/definitions/repository'
+        author: 
+          $ref: '#/components/definitions/user'
+```
+
+As references to `operationId` may not be possible (the `operationId` is an optional value), references may also be made through a relative `href`:
+
+```yaml
+components:
+  links:
+    UserRepositories:
+      # returns array of '#/components/definitions/repository'
+      href: '/2.0/repositories/{$response.username}'
+```
+
+or an absolute `href`:
+
+```yaml
+components:
+  links:
+    UserRepositories:
+      # returns array of '#/components/definitions/repository'
+      href: 'https://na2.gigantic-server.com/2.0/repositories/{$response.username}'
+```
+
+
+### Link Parameters
+Using the `operationId` to reference an operation in the definition has many benefits, including the ability to define media-type options, security requirements, response and error payloads.  Many operations require parameters to be passed, and these may be dynamic depending on the response itself.
+
+To specify parameters required by the operation, we can use a **Link Parameters Object**.  This object contains parameter names along with static or dynamic valus:
```

Comment:
The [2.0 specification says](http://swagger.io/specification/#parameterObject):

> A unique parameter is defined by a combination of a name and location.

The same is in the [current state of the 3.0 version](https://github.com/OAI/OpenAPI-Specification/blob/OpenAPI.next/versions/3.0.md#parameterObject).

While when designing an API such things should be avoided, they can happen when describing an existing API, and the wording sounds like that was intentional. (I was not there when this was decided, so I don't know.)